doc/COMPILING/COMPILING-CMAKE-VCPKG.md

```diff
@@ -0,0 +1,48 @@
+# Disclaimer
+
+**WARNING**: CMake build is **NOT** official and should be used for *dev purposes ONLY*.
```

Comment:
No corresponding doc for the cross build?

Reply:
The cross builds are shown from `--list-presets` argument to `cmake` command. It displays what you are allowed to build based on the system you run it from (see "condition" in `CMakePresets.json`).
For instance, from Linux it will show you `"windows-tiles-sounds-x64" - x64 Release`.